src/platforms/windows/application/core_application_windows.cpp

```
@@ -0,0 +1,46 @@
+
+#include "application.hpp"
+
+namespace toy::application {
+
+uint32_t CoreApplication::pid() const noexcept {
+  static_assert(sizeof(uint32_t) == sizeof(DWORD), "uint32_t and DWORD must be the same size for bit_cast");
+
+  return bit_cast<uint32_t>(GetCurrentProcessId());
+}
+
+void CoreApplication::sleep(size_t milliseconds) const noexcept {
```

Cppcheck (2.20.0)

[style] The function 'argument' is never used.

(unusedFunction)